src/components/Todo.js

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
-import EditButton from './todo_action_buttons/EditButton';
-import ProgressButton from './todo_action_buttons/ProgressButton';
+
 import CompleteButton from './todo_action_buttons/CompleteButton';
 import DeleteButton from './todo_action_buttons/DeleteButton';
+import EditButton from './todo_action_buttons/EditButton';
+import ProgressButton from './todo_action_buttons/ProgressButton';
```

Comment:
One of advantages of well organized code is gathering all relevant component/functionality and passing all imports through /index.js file. You can try creating an index.js file in each of /components subdirectory and re-export all components from there, so you'll have organized imports (sometime oneliners). You will have smth like this `import { CompleteButton, DeleteButton, DeleteButton, ProgressButton } from './todo_action_buttons'`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export

Reply:
fixed